leader-worker-set/rockcraft.yaml

```diff
@@ -0,0 +1,52 @@
+# Source https://github.com/kubernetes-sigs/lws/blob/v0.7.0/Dockerfile
+name: leader-worker-set
+summary: An image for LeaderWorkerSet
+description: |
+  LeaderWorkerSet: An API for deploying a group of pods as a unit of replication.
+  It aims to address common deployment patterns of AI/ML inference workloads,
+  especially multi-host inference workloads where the LLM will be sharded and run across multiple devices on multiple nodes.
+version: "0.7.0"
+license: Apache-2.0
+base: ubuntu@22.04
+
+platforms:
+  amd64:
+
+services:
+  kubeflow-trainer:
```

Comment:
nitpick: maybe we can call the Pebble service like this same rock, to avoid confusion?